src/VisualStudio/Core/Def/Implementation/MoveToNamespace/MoveToNamespaceDialog.xaml

```diff
@@ -0,0 +1,99 @@
+﻿<vs:DialogWindow x:Uid="MoveToNamespaceDialog"
```

Comment:
(We discussed actual UI stuff offline, so I'll refrain from too many comments here.)